clusterous/cluster.py

```diff
@@ -1088,6 +1370,7 @@ def docker_image_info(self, image_name_str):
         ssh = self._ssh_to_controller()
         ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
         ssh.connect(hostname = self._get_controller_ip(),
+                    port = 22000,
```

Comment:
I realised that this ssh.connect is redundant and should be removed. _ssh_to_controller already creates and returns the connection. So the port parameter should go in that method. Regarding the names of the methods "_ssh_to_controller" can remain what it is (since we are effectively connecting to the controller), but "_get_controller_ip" should change to "_get_nat_ip".